test/plugin_helper/test_child_process.rb

```diff
@@ -335,12 +335,12 @@ def configure(conf)
 
   test 'can execute external command many times, with leading once executed immediately' do
     ary = []
-    arguments = ["-e", "puts 'okay'; STDOUT.flush rescue nil"]
+    arguments = ["okey"]
     Timeout.timeout(TEST_DEADLOCK_TIMEOUT) do
```

Comment:
This change okay => okey intentionally? ( to distingish test case ?)

Reply:
No, I'll fix it.

Reply:
Fixed.